### ISSUE TITLE
Denoise and Latent mode / Poisson

### ISSUE THREAD

**Issue by inakicm** [theislab/dca#16]
Hi,

Thanks a lot for the paper and the package. I'm new to neural network.

I have a few questions:

1. When mode='latent', we are able to get the bottleneck layer which represent latent representation of cells. But is it possible to get also the matrix of the encoder layers and also the decoder layers? And also the W parameters for each layer?

2. I saw that you have implemented the Poisson Autoencoder (with Poisson as loss function) in the package but you did not talk about it in your paper. Did you perform some test with it? What kind of results did you get?

Thanks again,
Best,
Inaki
